lib/plugins/aws/tests/validate.js

```diff
@@ -49,6 +49,17 @@ describe('#validate()', () => {
     });
   });
 
+  it('should default to AWS_DEFAULT_REGION region if region is not provided ', () => {
+    awsPlugin.options.region = false;
+    process.env.AWS_DEFAULT_REGION = 'us-west-2';
```

Comment:
Hi there,
Thanks for putting this change in, as I'll be one of the ones benefitting from it :)
Just a note that I believe the existing test on us-east-1 will start failing if the user has AWS_DEFAULT_REGION set when running the test suite, as validate() will pick up on it and set it as the new region.
Maybe in that test you just need to make sure AWS_DEFAULT_REGION has been unset first.
HTH
Daniel